app/controllers/result_filters/location_controller.rb

```diff
@@ -11,13 +12,17 @@ def start
     end
 
     def create
+      sanitized_params = filter_params.
```

Comment:
We should strip out non-ASCII characters before redirecting to the ProviderController